spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -212,14 +212,15 @@ data class SpdxLicenseIdExpression(
         val id: String,
         val anyLaterVersion: Boolean = false
 ) : SpdxExpression() {
+    private val license = SpdxLicense.forId(toString())
```

Comment:
I would prefer to call this property `spdxLicense` or `spdxId`, otherwise having `id` and `license` could be confusing.

Reply:
Done.